src/auditwheel/wheeltools.py

```diff
@@ -223,9 +224,8 @@ def add_platforms(
         out_dir = "."
         wheel_fname = basename(wheel_ctx.in_wheel)
 
-    parsed_fname = WHEEL_INFO_RE.match(wheel_fname)
-    fparts = parsed_fname.groupdict()
-    original_fname_tags = fparts["plat"].split(".")
+    wheel_components = parse_wheel_filename(wheel_fname)
```

Comment:
I would make this consistent with `main_addtag.py` and unpack the tuple in the assignment as well..